java/arcs/core/data/Capabilities.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data
+
+/** A class containing a set of capabilities describing a store. */
+data class Capabilities(val capabilities: Set<Capability>) {
+    fun isEmpty(): Boolean = capabilities.isEmpty()
+
+    fun contains(other: Capabilities): Boolean = capabilities.containsAll(other.capabilities)
+
+    val isPersistent: Boolean
+        get() = capabilities.contains(Capability.Persistent)
+    val isTiedToRuntime: Boolean
+        get() = capabilities.contains(Capability.TiedToRuntime)
+    val isTiedToArc: Boolean
+        get() = capabilities.contains(Capability.TiedToArc)
```

Comment:
Re-iterating prior comment: KDoc describing what these are/mean would be very valuable.

Reply:
sorry, my bad, i didn't read the original comment carefully enough, thought it was only referring to class level comment. added now.